bookshelf/php.ini

```diff
@@ -0,0 +1 @@
+extension=mongodb.so
```

Comment:
I would prefer us support the `ext-mongodb` directive in `composer.json` rather than enable the extension by default. But yeah, this is a conversation for `php-docker`